lessons/Early-Math/evaluating-exponents/langs/en-us/pages/exponents-rubric.adoc

```diff
@@ -0,0 +1,41 @@
+[.landscape]
+= Rubric: Evaluating Exponents
```

Comment:
What is this page for? It's not linked or referenced from anywhere else in the lesson.

Reply:
I mentioned in the PR message that it didn't feel worth creating a project page for this project, because the project wasn't so great. We typed some project instructions into the ppsd scope and sequence, but I'm just gonna delete them.